webrtc_live_view/lib/webrtc_live_view_web/live/home.ex

```diff
@@ -0,0 +1,47 @@
+defmodule WebrtcLiveViewWeb.Live.EchoLive do
+  use WebrtcLiveViewWeb, :live_view
+
+  alias Membrane.WebRTC.Live.{Capture, Player}
+
+  def mount(_params, _session, socket) do
+    socket =
+      if connected?(socket) do
```

Comment:
When can it happen, that the socket is not connected in `mount`?

Reply:
`mount/3` is always called 2 times, the first time when the socket is not connected, the second time when it is connected